lib/iris/tests/unit/constraints/test_NameConstraint.py

```diff
@@ -6,16 +6,17 @@
 """Unit tests for the `iris._constraints.NameConstraint` class."""
 
 from __future__ import absolute_import, division, print_function
-from six.moves import filter, input, map, range, zip  # noqa
-
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests
 
 from unittest.mock import Mock, sentinel
 
+from six.moves import filter, input, map, range, zip  # noqa
+
 from iris._constraints import NameConstraint
 
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests  # isort:skip
```

Comment:
Needs moving? Not being imported first.

Reply:
Indeed... and this file also had references to `__future__` and `six`... tsk. Fixed that also :+1: